static/src/javascripts/projects/common/modules/article/spacefinder.js

```diff
@@ -175,7 +175,7 @@ define([
         var body;
 
         rules = rules || defaultRules;
-        body = rules.bodySelector ? document.querySelector(rules.bodySelector) : document;
+        body = (rules.bodySelector && document.querySelector(rules.bodySelector)) || document;
```

Comment:
Cheers! But I think there might be something else leading to it. If *the spacefinder is running* (saying this I feel like Mr Spock) that means the JS is either trying to insert adverts or rich links in the page. Both may be looking for a `.js-article__body` instead, so you might want to disable them in The Minute templates.

Great work by the way, it's looking hot!